scripts/drone_publish_main.sh

```diff
@@ -1,15 +1,53 @@
 #!/usr/bin/env sh
 
+dst="${DESTINATION}/${DRONE_BUILD_EVENT}"
+local_dst="file://dist/${DRONE_BUILD_EVENT}"
+set -e
+
 # This command enables qemu emulators for building Docker images for arm64/armv6/armv7/etc on the host.
 docker run --privileged --rm tonistiigi/binfmt --install all
 
-dagger run go run ./cmd \
+# Build all of the grafana.tar.gz packages.
+echo "Building tar.gz packages..."
+dagger run --silent go run ./cmd \
   package \
+  --version=$(cat package.json | jq -r .version | sed s/pre/${DRONE_BUILD_NUMBER}/g)
+  --yarn-cache=${YARN_CACHE_FOLDER} \
   --distro=linux/amd64 \
   --distro=linux/arm64 \
+  --distro=windows/amd64 \
+  --distro=darwin/amd64 \
+  --checksum \
   --build-id=${DRONE_BUILD_NUMBER} \
   --grafana-dir=${GRAFANA_DIR} \
   --github-token=${GITHUB_TOKEN} \
   --go-version=${GO_VERSION} \
-  --destination=${DESTINATION}/${DRONE_BUILD_EVENT} \
-  --gcp-service-account-key-base64=${GCP_KEY_BASE64}
+  --version=${DRONE_TAG} \
```

Comment:
Hm, I think `DRONE_TAG` is not available to `main` builds? Maybe we need the same thing that goes in the `--version` arg here?

Reply:
yeah you're right, good catch. I need to change that.